### PR TITLE
fix cssrewrite

### LINKED ISSUE #393: CSS rewrite does not work
currently cssrewrite does not work it should be activated for css files

### PR DESCRIPTION
**Pull request by alexander-schranz**
__Tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes
- [ ] submit changes to the documentation
- [ ] added changelog line

__Informations:__

| Q             | A
| ------------- | ---
| Tests pass?   | none
| Fixed tickets | fixes #393
| BC Breaks     | none
| Doc           | none

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -30,7 +30,8 @@ assetic:
     #bundles:        []
     #java: /usr/bin/java
     filters:
-        cssrewrite: ~
+        cssrewrite:
+            apply_to: '\.(css)$'
```
Comment (dantleech) [inline app/config/config.yml]: So does this apply cssrewrite only to `*.css` files? What does it do currently?

Comment (alexander-schranz) [inline app/config/config.yml]: it fixes the relative paths for assetic:dump most projects are first developed on a static web where the images will not be in /bundles/... and sometimes a framework is used which generated relative paths.
It only applies to `css` should i add other formats? I think the assetic bundle also have support for `less` or `sass/scss` file generating.

Comment (dantleech) [inline app/config/config.yml]: Really can't comment here, as I don't know much about it.

Is this requirement change on a project-by-project basis?

Maybe @wachterjohannes or @danrot can comment

Comment (danrot) [inline app/config/config.yml]: Please show me some concrete examples on monday, I also don't know what this is about.

**Comment by wachterjohannes** [comment]
@alexander-schranz i have added a css file with a relative path to a background image and it works when i do `assetic:dump` is that the usage of this PR? then i can merge it!

**Comment by alexander-schranz** [comment]
@wachterjohannes yes :)

**Pull request merged**